packages/story-editor/src/components/canvas/layout.js

```diff
@@ -356,8 +356,8 @@ function useLayoutParamsCssVars() {
     '--viewport-height-px': `${viewportHeight}px`,
     '--overflow-x': hasHorizontalOverflow ? 'scroll' : 'visible',
     '--overflow-y': hasVerticalOverflow ? 'scroll' : 'visible',
-    '--scroll-left-px': `-${scrollLeft}px`,
-    '--scroll-top-px': `-${scrollTop}px`,
+    '--scroll-left-px': `${-scrollLeft}px`,
+    '--scroll-top-px': `${-scrollTop}px`,
```

Comment:
It was actually pretty obvious.

When scrolling horizontally in RTL, `scrollLeft` becomes negative, so this should have been a positive number, but ended up a double negative, which of course didn't work. Easy fix.

The fix for the `scrollTop` is actually irrelevant for now, but why not.